aten/src/ATen/native/cuda/AveragePool3d.cu

```diff
@@ -446,6 +475,13 @@ void avg_pool3d_backward_out_cuda_template(
   TORCH_CHECK((gradOutput.ndimension() == 4 || gradOutput.ndimension() == 5),
     "non-empty 4D or 5D (batch mode) tensor expected for gradOutput");
 
+  //if divisor=0 then we will ignore it
```

Comment:
same here, also put a space between `//` and the comment, makes it easier to read.

Reply:
changed